### PR TITLE
Add tune proto

### PR DESCRIPTION
**Pull request by coderkalyan**
Rather simple proto file, for the tune plugin...

Let me know if something should be changed.

### CODE REVIEW

File: protos/tune/tune.proto [hunk 1]
```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package mavsdk.rpc.geofence;
+
+option java_package = "io.mavsdk.tune";
+option java_outer_classname = "TuneProto";
+
+service TuneService {
```
Comment (JonasVautherin) [inline protos/tune/tune.proto]: Missing documentation for the service

File: protos/tune/tune.proto [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  /*
+   * Send a tune to be played by the system.
+   */
+  rpc PlayTune(PlayTuneRequest) returns(PlayTuneResponse) {}
+}
+
+enum SongElement {
```
Comment (JonasVautherin) [inline protos/tune/tune.proto]: Documentation for the enum and its possible values :sweat_smile:

File: protos/tune/tune.proto [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  NOTE_G = 15;
+  NOTE_PAUSE = 16;
+  SHARP = 17;
+  FLAT = 18;
+  OCTAVE_UP = 19;
+  OCTAVE_DOWN = 20;
+}
+
+message PlayTuneRequest {
+  repeated SongElement tune = 1;
```
Comment (JonasVautherin) [inline protos/tune/tune.proto]: This one requires documentation (but the `TuneResult` below doesn't)

**Comment by coderkalyan** [comment]
@JonasVautherin I added the comments you requested :smile:

File: protos/tune/tune.proto [hunk 4]
```diff
@@ -5,23 +5,26 @@ package mavsdk.rpc.geofence;
...
-  /*
-   * Send a tune to be played by the system.
-   */
+  // Send a tune to be played by the system.
   rpc PlayTune(PlayTuneRequest) returns(PlayTuneResponse) {}
 }
 
+// represents an element of the tune
 enum SongElement {
+  // playing styles
```
Comment (JonasVautherin) [inline protos/tune/tune.proto]: Hehe I'm sorry about that, but the parser won't work with it :sweat_smile:. It has to be after each element, e.g.:

```
STYLE_LEGATO = 0; // After this element, start playing legato
STYLE_NORMAL = 1; // After this element, start playing normal
STYLE_STACCATO = 2; // After this element, start playing staccato
```

I understand it's a bit annoying, but that it then used to generate the API documentation directly in the code (currently only in Python) :+1:.

**Comment by coderkalyan** [comment]
@JonasVautherin no worries, I changed the comment format to after each enum value.

**Pull request merged**